RELEASE-NOTES.txt

```diff
@@ -1,9 +1,15 @@
+1.21.0
+------
+* [Android] Reduced padding around text on Rich Text based blocks.
```

Comment:
There was an Android specific fix. But there was also a general change that reduced padding around text on Rich Text based blocks.